event_handler/sources.py

```diff
@@ -60,6 +60,7 @@ def pagerduty_verification(signatures, body):
         print(signatures)
         raise Exception("Pagerduty signature is empty")
     print(f" pagerduty signature :: {signatures.get('X-Pagerduty-Signature')} ")
+    pagerduty_signature = signatures.get('X-Pagerduty-Signature')
```

Comment:
Looks fine from what we've seen thus far, since the headers have only included one signature. The [documentation](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTkz-verifying-signatures#verifying-the-signature) suggests there could be more than one signature though, which is why I think it was the way it was.

Reply:
Fait comment, The code around signature needs to be changed and tidying up.. i have just added for quick test to get a clear channel... we would address the possibility of two tokens if in case.